### PR TITLE
Refactor props in crossgen.target

### PR DESCRIPTION
**Pull request by jgoshi**
@piotrpMSFT @livarcocc @krwq @jonsequitur @eerhardt

### CODE REVIEW

**Comment by jgoshi** [comment]
test OpenSUSE13.2 x64 Debug Build

File: build/ToolInfo.props [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<Project>
```
Comment (piotrpMSFT) [inline build/ToolInfo.props]: This file feels like it's a bit overloaded. NuGetPackagesDir should probably reside in dir.props directly, or in a NuGet-specific file. The remainder describes the Crossgen tool explicitly.

File: dir.props [hunk 2]
```diff
@@ -21,13 +21,11 @@
   <Import Project="build/BuildDefaults.props" />
   <Import Project="build/BuildOutputs.props" />
   <Import Project="build/Stage0.props" />
+  <Import Project="build/ToolInfo.props" />
   <Import Project="build/VersionBadge.props" />
   <Import Project="build/Microsoft.DotNet.Cli.BundledRuntimes.props" />
   
   <PropertyGroup>
-    <NuGetPackagesDir>$(NUGET_PACKAGES)</NuGetPackagesDir>
```
Comment (piotrpMSFT) [inline dir.props]: Yea, I'd undo this change [NuGetPackagesDir]. It's so central and impactful, I think it belongs where it was.

**Comment by jgoshi** [comment]
test Debian8.2 x64 Debug Build
